src/view/com/composer/photos/SelectPhotoBtn.tsx

```diff
@@ -23,12 +23,12 @@ export function SelectPhotoBtn({gallery}: Props) {
   const onPressSelectPhotos = useCallback(async () => {
     track('Composer:GalleryOpened')
 
-    if (!isDesktopWeb && !(await requestPhotoAccessIfNeeded())) {
+    if (!isNative && !(await requestPhotoAccessIfNeeded())) {
```

Comment:
```suggestion
    if (isNative && !(await requestPhotoAccessIfNeeded())) {
```

Reply:
Sorry, I didn't notice the problem here. I recently contracted COVID-19, which has left me a bit out of sorts.

Reply:
No problem -- please get better!